src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminListFilters.cshtml

```diff
@@ -1,12 +1,47 @@
 @model ContentOptionsViewModel
 @{
-    var formActionWithoutTypeName = Url.RouteUrl(new { area = "OrchardCore.Contents", controller = "Admin", action = "List", contentTypeId = "" });
+    var ownerFilterUserSearchUrl = Url.RouteUrl(new { area = "OrchardCore.Contents", controller = "Admin", action = "OwnerFilterUserSearch" });
 }
 <div class="btn-group filter">
     @if (Model.ContentTypeOptions.Count > 0)
     {
         <select asp-for="SelectedContentType" asp-items="Model.ContentTypeOptions" class="selectpicker contenttypes-selectpicker show-tick me-2" data-header="@T["Filter by content type"]" data-selected-text-format="static" data-width="fit" data-dropdown-align-right="auto" title="@T["Content Type"]" data-style="btn-sm" data-live-search="true"></select>
     }
     <select asp-for="ContentsStatus" asp-items="@Model.ContentStatuses" class="selectpicker contentstatuses-selectpicker show-tick me-2" data-header="@T["Filter by status"]" data-selected-text-format="static" data-width="fit" data-dropdown-align-right="auto" title="@T["Show"]" data-style="btn-sm"></select>
+    <select asp-for="SelectedOwnerUserName" class="ajax-selectpicker contentowners-selectpicker show-tick me-2" data-header="@T["Filter by owner"]" data-selected-text-format="static" data-width="fit" data-dropdown-align-right="auto" title="@T["Owner"]" data-style="btn-sm" data-live-search="true"></select>
     <select asp-for="OrderBy" asp-items="@Model.ContentSorts" class="selectpicker contentsorts-selectpicker show-tick" data-header="@T["Sort by"]" data-width="fit" data-selected-text-format="static" data-dropdown-align-right="true" title="@T["Sort"]" data-style="btn-sm"></select>
 </div>
+
+<script at="Foot">
+    $(function () {
```

Comment:
Are you able to use vanilla JS instead of jQuery? and if you need to depend on jQuery, you'll need to add add dependency to jquery on line 15.

Reply:
Unfortunately bootstrap-select depends on jQuery. I added the dependency. Its parent view is where it is importing bootstrap-select. Should there be a dependency on that too?